demo/src/components/Main.js

```diff
@@ -347,6 +351,8 @@ export default class Main extends Component {
             className={'PreviewArea'}
             disabledDates={[new Date(), addDays(new Date(), 3)]}
             minDate={addDays(new Date(), -3)}
+            cellInfo={cellInfo}
+            cellInfoClassName="abc"
```

Comment:
What do you think about instead of passing a list of `date` & `value`, passing a function when called with a date, returns the extra info to be shown?